x-pack/test/fleet_api_integration/apis/download_sources/crud.ts

```diff
@@ -133,7 +130,7 @@ export default function (providerContext: FtrProviderContext) {
         expect(downloadSource.host).to.eql('https://test.co:403');
       });
 
-      it('should allow to update an existing download source', async function () {
+      it('should allow to update is_default for existing download source', async function () {
```

Comment:
`error  Test title is used multiple times in the same test suite  mocha/no-identical-title`